clientd3d/graphics.c

```diff
@@ -290,10 +302,11 @@ int GetMSDrawFrame(void)
 void RedrawForce(void)
 {
    HDC hdc;
-   static DWORD lastEndFrame = 0;
-   DWORD endFrame, startFrame;
-   int totalFrameTime, oldMode;
+   int oldMode;
    char buffer[32];
+   
+   steady_clock_time_point endFrame, startFrame;
+   std::chrono::duration<double> elapsedTime;
```

Comment:
I think you can declare most of these at the point of first use?  oldMode for sure.